<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>writeData</class>
 <widget class="QWidget" name="writeData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_goBack_writeNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/back.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtToWrite">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>691</width>
     <height>311</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start_sort">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>430</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(184, 184, 184);</string>
   </property>
   <property name="text">
    <string>Sort</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNameOfSort">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>9</y>
     <width>301</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tự nhập dữ liệu</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_goHome">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/home_icon.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>65</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vui lòng nhập dữ liệu bạn muốn sắp xếp</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_ready">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>490</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
